python/pfs/drp/stella/stellaLib.i

```diff
@@ -98,994 +98,58 @@ Interface to Stella
-
 %shared_ptr(pfs::drp::stella::FiberTraceSet<float, unsigned short, float>);
-%shared_ptr(pfs::drp::stella::FiberTraceSet<double, unsigned short, float>);
-
-//%shared_ptr(pfs::drp::stella::PSF<float>);
-//%shared_ptr(pfs::drp::stella::PSF<double>);
-
-//%shared_ptr(std::vector<PTR(pfs::drp::stella::PSF<float>)>);
-//%shared_ptr(std::vector<PTR(pfs::drp::stella::PSF<double>)>);
-
-//%shared_ptr(pfs::drp::stella::PSFSet<float>);
-//%shared_ptr(pfs::drp::stella::PSFSet<double>);
-
-//%shared_ptr(std::vector<PTR(pfs::drp::stella::PSFSet<float>)>);
-//%shared_ptr(std::vector<PTR(pfs::drp::stella::PSFSet<double>)>);
 
 %shared_ptr(pfs::drp::stella::Spectrum<float, unsigned short, float, float>);
-%shared_ptr(pfs::drp::stella::Spectrum<double, unsigned short, float, float>);
-%shared_ptr(pfs::drp::stella::Spectrum<float, unsigned int, float, float>);
-%shared_ptr(pfs::drp::stella::Spectrum<double, unsigned int, float, float>);
-
 %shared_ptr(std::vector<PTR(pfs::drp::stella::Spectrum<float, unsigned short, float, float>)>);
-%shared_ptr(std::vector<PTR(pfs::drp::stella::Spectrum<double, unsigned short, float, float>)>);
-%shared_ptr(std::vector<PTR(pfs::drp::stella::Spectrum<float, unsigned int, float, float>)>);
-%shared_ptr(std::vector<PTR(pfs::drp::stella::Spectrum<double, unsigned int, float, float>)>);
-
 %shared_ptr(pfs::drp::stella::SpectrumSet<float, unsigned short, float, float>);
-%shared_ptr(pfs::drp::stella::SpectrumSet<double, unsigned short, float, float>);
-%shared_ptr(pfs::drp::stella::SpectrumSet<float, unsigned int, float, float>);
-%shared_ptr(pfs::drp::stella::SpectrumSet<double, unsigned int, float, float>);
-
 %shared_ptr(std::vector<PTR(ndarray::Array<float, 2, 1>)>);
-%shared_ptr(std::vector<PTR(ndarray::Array<double, 2, 1>)>);
 
 %include "std_vector.i"
-// """Shared Pointers (!) to std::vectors"""
-//%template(SPVectorF) std::vector<float>;
-//%template(SPVectorUS) std::vector<unsigned short>;
-//%template(SPVectorUI) std::vector<unsigned int>;
-//%template(SPVectorUL) std::vector<unsigned long>;
-//%template(SPVectorI) std::vector<int>;
-//%template(SPVectorL) std::vector<long>;
-//%template(SPVectorD) std::vector<double>;
-//%template(VectorPVF) std::vector<PTR(std::vector<float>)>;
-
 %include "pfs/drp/stella/FiberTraces.h"
-%template(FTVectorF) std::vector<PTR(pfs::drp::stella::FiberTrace<float, unsigned short, float>)>;
-%template(FTVectorD) std::vector<PTR(pfs::drp::stella::FiberTrace<double, unsigned short, float>)>;
 
-//%include "pfs/drp/stella/PSF.h"
-//%template(PSFVectorF) std::vector<PTR(pfs::drp::stella::PSF<float>)>;
-//%template(PSFVectorD) std::vector<PTR(pfs::drp::stella::PSF<double>)>;
-//%template(PSFSetVectorF) std::vector<PTR(pfs::drp::stella::PSFSet<float>)>;
-//%template(PSFSetVectorD) std::vector<PTR(pfs::drp::stella::PSFSet<double>)>;
+%template(FTVectorF) std::vector<PTR(pfs::drp::stella::FiberTrace<float, unsigned short, float>)>;
```

Comment:
Oups, actually needed it to prevent a memory leak because SWIG couldn't find the destructor